R/read_redcap.R

```diff
@@ -170,6 +188,23 @@ read_redcap <- function(redcap_uri,
   # Check that results were returned
   check_redcap_populated(db_data)
 
+  # Check that requested args exist
```

Comment:
How about: 
`# Check whether DAGs or survey fields are present in the project, if requested`